gensim/models/doc2vec.py

```diff
@@ -209,10 +209,13 @@ class Doc2Vec(BaseWordEmbeddingsModel):
 
     """
     def __init__(self, documents=None, corpus_file=None, dm_mean=None, dm=1, dbow_words=0, dm_concat=0,
-                 dm_tag_count=1, docvecs=None, docvecs_mapfile=None, comment=None, trim_rule=None, callbacks=(),
-                 **kwargs):
+                 dm_tag_count=1, docvecs=None, docvecs_mapfile=None, comment=None, trim_rule=None, 
+                 pretrained_emb=None, 
```

Comment:
Put this after `callbacks`. We don't want to change the order of the existing parameters and break compatibility.

Reply:
accepted!